src/app/App.js

```diff
@@ -23,7 +23,7 @@ export default class AppProvider extends React.Component<PropsType> {
 		const { initialHtmlModeEnabled } = this.props;
 		let initialData = this.props.initialData;
 		let initialTitle = this.props.initialTitle;
-		if ( initialData === undefined ) {
+		if ( initialData === undefined && __DEV__ ) {
```

Comment:
@hypest seems this does the trick

Reply:
Yep, so just to be clear, doing so we disable running metro during our e2e tests and instead bundle a "production" version of the app (without minification to understand the stacktrace) that we install on our device when running local test (using `yarn test:e2e:android:local` for instance) or remotely on saucelabs using the circle ci script and running `yarn test:e2e:android`.